Find a functionally equivalent code:android.media.MediaFocusControl.dumpRCCStack:COMMENT
Method Modifier: private     
Comment:/**
 * Helper function:
 * Display in the log the current entries in the remote control stack, focusing
 * on RemoteControlClient data
 */

@@ -1,10 +1,9 @@
 {
     pw.println("\nRemote Control Client stack entries (last is top of stack):");
-    synchronized (mRCStack) {
-        Iterator<RemoteControlStackEntry> stackIterator = mRCStack.iterator();
+    synchronized (mPRStack) {
+        Iterator<PlayerRecord> stackIterator = mPRStack.iterator();
         while (stackIterator.hasNext()) {
-            RemoteControlStackEntry rcse = stackIterator.next();
-            pw.println("  uid: " + rcse.mCallingUid + "  -- id: " + rcse.mRccId + "  -- type: " + rcse.mPlaybackType + "  -- state: " + rcse.mPlaybackState + "  -- vol handling: " + rcse.mPlaybackVolumeHandling + "  -- vol: " + rcse.mPlaybackVolume + "  -- volMax: " + rcse.mPlaybackVolumeMax + "  -- volObs: " + rcse.mRemoteVolumeObs);
+            stackIterator.next().dump(pw, false);
         }
         synchronized (mCurrentRcLock) {
             pw.println("\nCurrent remote control generation ID = " + mCurrentRcClientGen);

